collectors/python.d.plugin/isc_dhcpd/README.md

```diff
@@ -41,11 +42,14 @@ Sample:
 
 ```yaml
 local:
-  leases_path       : '/var/lib/dhcp/dhcpd.leases'
-  pools       : '192.168.3.0/24 192.168.4.0/24 192.168.5.0/24'
+  leases_path: '/var/lib/dhcp/dhcpd.leases'
+  pools:
+    office:         '192.168.2.0/24'
+    wifi:           '192.168.3.10-192.168.3.20'
+    192.168.4.0/24: '192.168.4.0/24'
+    wifi-guest:     '192.168.5.0/24 192.168.6.10-192.168.6.20'
 ```
 
```

Comment:
we need to not about CIDR and IPRange formats here

Reply:
Now better?